packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -262,11 +298,25 @@ class DescopeWc extends BaseDescopeWc {
     return undefined;
   }
 
+  #subscribeStepState() {
+    this.stepState?.subscribe(
```

Comment:
if I understand correctly, we now subscribe twice - once for error state, and another time for anything else

Reply:
Yep